wallbox_client.py

```diff
@@ -327,16 +327,14 @@ def set_power_setpoint(self, charge_rate: int):
             charge_rate = 0
 
         # Clip values to min/max charging current
-        max_charging_power = self.args["wallbox_max_charging_power"]
-        max_discharging_power = self.args["wallbox_max_discharging_power"]
-        if charge_rate > max_charging_power:
+        if charge_rate > c.CHARGER_MAX_CHARGE_POWER:
```

Comment:
I prefer all constants being imported by name and avoid using abbreviations. 

so `from constants import CHARGER_MAX_CHARGE_POWER`

Reply:
We import about 20 (?) CONSTANTS, is this still the preferred method then?